src/components/Profile/components/Identification/Identification.module.scss

```diff
@@ -27,7 +27,7 @@ $image-side-width-desktop: 90px;
   box-shadow: 0px 2px 1px -1px $neutral-black-opacity20, 0px 1px 1px 0px $neutral-black-opacity15,
     0px 1px 3px 0px $neutral-black-opacity125;
   &_header {
-    background-color: $primary-blue;
+    background-color: var(--mui-palette-primary-300);
```

Comment:
I know this file is massive, but it seems to me that you ought to do an entire component in one go, rather than replacing it piece meal.

Reply:
Yep, most of the individual changes were just part of my experimentation, we're now going through whole components at once to change, and since this PR I've actually fully completed this component, so I'll include it in the next merge.